docs/UserGuide.md

```diff
@@ -1,10 +1,34 @@
 # User Guide
```

Comment:
Nice.... command table looks like it is formatted properly in the github website